src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -127,11 +131,15 @@ When setting `strict_min_version`, unless you’re targeting a specific version
 %}
 
 <!-- END: Single Column Body Module -->
+
 <!-- Single Column Body Module -->
 
 {% capture content %}
 
 ## Install and run your extension in Firefox for Android
+::: note alert
+Persistent loading is not yet supported on Firefox for Android 79+. These instructions are for temporary installation.
```

Comment:
Add another warning:

> Unlike `web-ext` for desktop and Fennec, the add-on will be loaded in the main browser profile instead of a new temporary profile directory.

(maybe with some copy editing)

Reply:
Loading in the main profile seems like a bug: https://github.com/mozilla/web-ext/issues/1950